bin/collator/src/parachain/service.rs

```diff
@@ -1285,84 +628,44 @@ where
         relay_chain_slot_duration: Duration::from_secs(6),
         proposer: cumulus_client_consensus_proposer::Proposer::new(proposer_factory),
         collator_service,
-        authoring_duration: Duration::from_millis(1500),
+        authoring_duration: Duration::from_millis(2000),
         reinitialize: false,
-    });
+    };
+
+    let fut = async move {
+        wait_for_aura(client).await;
+        aura::run::<Block, AuraPair, _, _, _, _, _, _, _, _, _>(params).await
+    };
```

Comment:
I see we'll be doing this for all runtimes, not just Shiden - I guess there's no harm in doing it while it drastically simplifies the code - is that the motivation?

Reply:
yes

Reply:
just delaying to start of block proposer until aura is present. This will have effect only on shiden